web/src/hooks/usePagination.ts

```diff
@@ -30,5 +33,14 @@ export default function usePagination() {
     history.replace(`${location.pathname}?page=${page}&pageSize=${pageSize}`);
   };
 
-  return { paginationConfig, savePageList };
+  const checkPageList = (ref: MutableRefObject<ActionType | undefined>) => {
+    const { current, pageSize, total } = ref.current!.pageInfo as PageInfo;
```

Comment:
```suggestion
    const { current, pageSize, total } = ref.current?.pageInfo as PageInfo;
```

`!.` will just remove `undefined` and `null` from the `current`'s type. Maybe `?.` is better?